unit-tests/func/MUR/l500/test-turn-off-IR-Reflectivity.cpp

```diff
@@ -0,0 +1,53 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2021 Intel Corporation. All Rights Reserved.
+
+//#test:tag max-usable-range
+//#test:device L500*
+
+#include "../../func-common.h"
+
+// Test group description:
+//       * This tests group verifies depth sensor max usable range option restrictions,
+//         according to [RS-8358]
+//
+// Current test description:
+//       * Verify turning off MUR option turns off IR Reflectivity option
```

Comment:
Explaiin, please, that reflectivity requires MUR to be enabled otherwise it cannot work.

Reply:
👍